pkg/route/registry/route/etcd/etcd_test.go

```diff
@@ -52,7 +53,7 @@ func validRoute() *api.Route {
 }
 
 func TestCreate(t *testing.T) {
-	storage, server := newStorage(t, &testAllocator{})
+	storage, server := newStorage(t, nil)
```

Comment:
@ramr I went ahead and changed this to nil as well to ensure we have a test with nil and a test with an allocator for both create and update